test/e2e/integration_runtime_test.go

```diff
@@ -135,22 +135,53 @@ func TestHostSocketConnect(t *testing.T) {
 	}
 	defer unix.Close(tmpDirFD)
 	// Use /proc/self/fd to generate path to avoid EINVAL on large path.
-	l, err := net.Listen("unix", filepath.Join("/proc/self/fd", strconv.Itoa(tmpDirFD), "test.sock"))
+	socketPath := filepath.Join("/proc/self/fd", strconv.Itoa(tmpDirFD), "test.sock")
+	l, err := net.Listen("unix", socketPath)
 	if err != nil {
 		t.Fatalf("listen error: %v", err)
 	}
 	defer l.Close()
+	// change the socket's permission so that "nonroot" can connect to it
```

Comment:
nit: All comments are full sentences. Capitalize the first word and period (.) at the end.